sktree/tree/_classes.py

```diff
@@ -925,59 +930,52 @@ def _build_tree(
             self.classes_ = self.classes_[0]
 
 
-class PatchObliqueDecisionTreeClassifier(DecisionTreeClassifier):
-    """A oblique decision tree classifier that operates over patches of data.
-
-    A patch oblique decision tree is also known as a manifold oblique decision tree
-    (called MORF in :footcite:`Li2019manifold`), where the splitter is aware of
-    the structure in the data. For example, in an image, a patch would be contiguous
-    along the rows and columns of the image. In a multivariate time-series, a patch
-    would be contiguous over time, but possibly discontiguous over the sensors.
-
+class ObliqueDecisionTreeRegressor(DecisionTreeRegressor):
+    """A decision tree Regressor.
+    Read more in the :ref:`User Guide <sklearn:tree>`. The implementation follows
+    that of :footcite:`breiman2001random` and :footcite:`TomitaSPORF2020`.
```

Comment:
Can you fix the spacing in the docstrings as I have suggested in the `_supervised_forest.py` file?